### PR TITLE
fix bug 363

### PR DESCRIPTION
**Pull request by ux-AylonS**
Signed-off-by: AylonS <aylons@reblaze.com>

### CODE REVIEW

File: src/doc-editors/RoutingProfilesEditor.vue [hunk 1]
```diff
@@ -301,10 +301,10 @@
                                     <select class="select is-small new-edge-function-selection"
                                             title="Edge Function ID"
                                             v-model="edgeFunctionMapEntryId">
-                                      <option v-for="edgeFunction in newEdgeFunctions(mapEntry.cloud_functions)"
```
Comment (Aviv-Galmidi) [inline src/doc-editors/RoutingProfilesEditor.vue:304]: We should display only new edge functions in the dropdown, not those which are already connected

File: src/doc-editors/RoutingProfilesEditor.vue [hunk 2]
```diff
@@ -301,10 +301,10 @@
...
+                                      <option v-for="edgeFunction in edgeFunctions"
                                               :key="edgeFunction.id"
                                               :value="edgeFunction.id">
-                                        {{ edgeFunction.name + ' ' + edgeFunction.description }}
```
Comment (Aviv-Galmidi) [inline src/doc-editors/RoutingProfilesEditor.vue:307]: Why did you add a dash between the name and description? For readability?

If so, let's also add it in Security Policy where we list Rate Limits in the same fashion

Comment (ux-AylonS) [reply src/doc-editors/RoutingProfilesEditor.vue:307]: Yes, readability. I will add it to Security Policy now.

**Review by Aviv-Galmidi — APPROVED** [review]
